tests/Functional/starWarsSchema.php

```diff
@@ -4,9 +4,50 @@
 
 use Digia\GraphQL\Relay\ArrayConnectionBuilder;
 use Digia\GraphQL\Relay\ConnectionArguments;
+use Digia\GraphQL\Relay\Node;
 use Digia\GraphQL\Relay\StoreConnectionBuilder;
 use function Digia\GraphQL\buildSchema;
 
+function getNodeId(string $type, $variable) {
+    if (\is_object($variable) && method_exists($variable, 'getId')) {
+        return Node::toGlobalId($type, $variable->getId());
+    }
+
+    if (\is_array($variable)) {
+        return Node::toGlobalId($type, $variable['id']);
+    }
+
+    throw new \RuntimeException('Unable to get an ID from the variable');
+}
+
+function addType(string $type, $variable) {
+    if (\is_array($variable)) {
+        $variable['type'] = $type;
+
+        return $variable;
+    }
+
+    throw new \RuntimeException('Unable to set a type on the variable');
+}
+
+function nodeResolver($root, $args)
```

Comment:
Since `$root` is not used in this method we could skip taking it as a parameter IMO